plugins/modules/ec2_instance.py

```diff
@@ -244,6 +244,10 @@
       - Whether to enable termination protection.
       - This module will not terminate an instance with termination protection active, it must be turned off first.
     type: bool
+  hibernation_option:
+    description:
```

Comment:
I would probably reference the link for the prerequisites for this option https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/hibernating-prerequisites.html

Reply:
It would be beneficial to add some checks for some requirements, like volume_type, encryption etc.